contracts/azorius/Azorius.sol

```diff
@@ -259,8 +258,6 @@ contract Azorius is Module, IAzorius {
      * @param _strategies array of BaseStrategy contract addresses to enable
      */
     function _setUpStrategies(address[] memory _strategies) internal {
-        if (strategies[SENTINEL_STRATEGY] != address(0))
-            revert AlreadySetupStrategies();
```

Comment:
Removed this check since it is impossible for this function to be called a second time anyways